mmf/trainers/core/evaluation_loop.py

```diff
@@ -25,6 +26,7 @@ def evaluation_loop(
             self.model.eval()
             disable_tqdm = not use_tqdm or not is_master()
             combined_report = None
+            metsumm("Before Validation Start:")
```

Comment:
this should go away, it's a debugging artifact. should not live in mmf codebase.